app/views/users/show.html.erb

```diff
@@ -125,9 +28,25 @@
           </div>
         <% end %>
       </div>
+      
+         <!-- Form for creating a new post -->
+
+      <%= form_with(model: @post, url: user_posts_path(@user), class: "form") do |form| %>
+        <div class="field">
+          <%= form.label :title %>
+          <%= form.text_field :title %>
+        </div>
+        <div class="field">
+          <%= form.label :text %>
+          <%= form.text_area :text %>
+        </div>
+        <div class="actions">
+          <%= form.submit 'Create Post' %>
+        </div>
+      <% end %>
+
       <div class="see-all-posts">
-        <%= link_to "View all posts", user_posts_path(@user) %>
+        <%= link_to "See all posts", user_posts_path(@user) %>
       </div>
     </div>
```

Comment:
- [ ] Great job adding forms to your app. However, I am unable to see the list of posts created by a user when I did so. A user should be able to see list of posts when clicking on the `See all posts` link. This feature is not functioning as expected at this moment. Kindly look into it and fix.

### App Behaviour

[post-error.webm](https://github.com/BenMKT/BlogApp/assets/26597432/d9ad1a88-fb9e-401d-8242-fc1d14dc8b30)

- [ ] As a result of my inability to see the user posts, I cannot go further to test for the comments for a post and the likes for a post. Kindly check these two requirements too before asking for a new review. Good luck.

Reply:
- Refactored code base to resolve bugs.